benchmark-scripts/format_avc_mp4.sh

```diff
@@ -59,7 +65,7 @@ then
 	exit 1
 fi
 
-result=${1/.mp4/"-$WIDTH-$FPS-bench.mp4"}
+result="$INDEX-"${1/.mp4/"-$WIDTH-$FPS-bench.mp4"}
```

Comment:
```suggestion
result="$INDEX-"${1/.mp4/"-$WIDTH-$FPS-bench.mp4"}
```
if $INDEX="" empty, this looks weird. we should include the `-` into INDEX variable.

Reply:
yeah I just noticed this too and fixed it.